File: pythonlibrary/aibrd/parsers/markdown.py
from ..models.brd import RawBRD


def parse_markdown(file_path: str) -> RawBRD:
    with open(file_path, encoding="utf-8") as f:
        text = f.read()
    return RawBRD(text=text, source=file_path, file_type="markdown")

File: vscodebase/src/core/parsers/markdown.ts
import * as fs from 'fs'
import { RawBRD } from '../models/brd'

export function parseMarkdown(filePath: string): RawBRD {
  const text = fs.readFileSync(filePath, 'utf-8')
  return {
    text,
    source: filePath,
    fileType: 'markdown'
  }
}
